registry.json

```diff
@@ -1787,5 +1787,20 @@
     "telegram":"manizzler",
     "operatorName":"manizzler",
     "github":"manizzle"
+  },
+     {
+    "ticker": "SBLYR",
+    "poolId": "8264de3cdb1798dd8758e24cda5101184b44543e7c4421c7815f9ed8",
+    "social": {
+      "twitter": "sublayerio",
+      "facebook": "",
+      "reddit": "",
+      "instagram": "",
+      "website": "https://pool.sublayer.io",
+      "youtube": ""
+    },
+    "telegram": "",
+    "operatorName": "PiTrippy",
```

Comment:
If you don't provide a real first name or full name, please leave this field empty.